src/client/src/components/pages/dashboard/index.jsx

```diff
@@ -39,9 +48,16 @@ class index extends Component {
   };
   
   render() {
+    const { employed, notEmployed} = this.state;
     return (
       <React.Fragment>
         <Header value="Dashboard" />
+        <PieChart
+          sections={[
+            { title: "employed", percentage: employed},
+            { title: "not employed", percentage: notEmployed},
+        ]}
+      /> 
         <div className="cards">
           <Link className="static-count" to="/participants/view">
             <Statistics number={this.state.participant} value="Participant" />
```

Comment:
use destructure object for (participant, course, worker )

```
const { participant,course,worker,employed,notEmployed }  = this.state
```

Reply:
done :ok_hand: